site-src/performance/regression-testing/index.md

```diff
@@ -0,0 +1,114 @@
+# Regression Testing
+
+Regression testing ensures recent code changes do not negatively impact the performance and stability of the Inference Gateway.
+
+This guide demonstrates how to run regression testing against the Gateway API inference extension. Benchmarks are conducted using the [Latency Profile Generator](https://github.com/AI-Hypercomputer/inference-benchmark) (LPG) to simulate traffic and collect detailed metrics.
+
+## Prerequisites
```

Comment:
There is a lot of duplicate from https://gateway-api-inference-extension.sigs.k8s.io/performance/benchmark/.

Can we restructure this doc to focus more on defining the regression test cases, and then refer to the benchmark doc on how to perform the benchmarks